### PR TITLE
Collinsc2

### PR DESCRIPTION
**Pull request by collinsc2**
merge pls

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 import random
 from time import sleep
 
-delay = 1.0          # change to 0.0 for testing/speed runs; larger for dramatic effect!
```
Comment (sheggen) [inline t04_refactored.py:24]: No code changes outside your function. Please revert these.

File: t04_refactored.py [hunk 2]
```diff
@@ -53,11 +53,12 @@ def end_story(user):
     :param user: the user's name
     :return: None
     """
-    print("Congratulations, " + user + ", you have made it to the end of this... strange... adventure. I hope you feel accomplished.")
```
Comment (sheggen) [inline t04_refactored.py:56]: No code changes outside your function. Please revert these.

File: t04_refactored.py [hunk 3]
```diff
@@ -53,11 +53,12 @@ def end_story(user):
...
+    print(
+        "Congratulations, " + user + ", you have made it to the end of this... strange... adventure. I hope you feel accomplished.")
     print()
     print()
     print()
-    sleep(delay*5)
```
Comment (sheggen) [inline t04_refactored.py:60]: No code changes outside your function. Please revert these.

File: t04_refactored.py [hunk 4]
```diff
@@ -376,8 +380,59 @@ def team_20_adv():
...
 def team_21_adv():
     pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/16WddE8AhpLg8duTmMrZPy8rBFTPxWAWdfpYzfPK1m_w/edit?usp=sharing
+    Cole Collins
+    """
+
+
+print("\n")
```
Comment (sheggen) [inline t04_refactored.py:389]: All of this code should be indented inside your team's  function

File: t04_refactored.py [hunk 5]
```diff
@@ -376,8 +380,59 @@ def team_20_adv():
...
+    sleep(delay)
+    if number >= 13:
+        print("Congrats! The snake decided to spare you!")
+        dead = False
+        sleep(delay)
+    else:
+        print("You chose a wrong number, which angers the snake. He and the other snakes and spiders bite you!")
 
+#
+if dead:
```
Comment (sheggen) [inline t04_refactored.py:433]: duplicate conditional for if dead.

**Comment by collinsc2** [comment]
Updated the code

File: t04_refactored.py [hunk 6]
```diff
@@ -427,14 +430,63 @@ def team_19_adv():
...
 
+
 def team_20_adv():
     pass
     # TODO Add your code here
 
 
 def team_21_adv():
+    global dead
     pass
```
Comment (sheggen) [inline t04_refactored.py:441]: Remove this placeholder text

File: t04_refactored.py [hunk 7]
```diff
@@ -427,14 +430,63 @@ def team_19_adv():
...
+        number = int(float(input("Which number do you pick? (only put whole numbers): ")))
+        sleep(delay)
+        if number >= 13:
+            print("Congrats! The snake decided to spare you!")
+            dead = False
+            sleep(delay)
+        elif number < 13:
+            print("You chose a wrong number, which angers the snake. He and the other snakes and spiders bite you!")
+        else:
+            print("Oh no! You died! Try again by hitting the green play button.")
```
Comment (sheggen) [inline t04_refactored.py:490]: `kill_if_dead()`: function should be used to kill your user

**Comment by collinsc2** [comment]
Fixed pass and added kill_if_dead()

File: t04_refactored.py [hunk 8]
```diff
@@ -480,14 +483,63 @@ def team_19_adv():
...
+
 def team_20_adv():
     pass
     # TODO Add your code here
 
 
 def team_21_adv():
-    pass
-    # TODO Add your code here
+    global dead
```
Comment (sheggen) [inline t04_refactored.py:493]: No code should come before the docstring (or PyCharm can't use the docstring)

File: t04_refactored.py [hunk 9]
```diff
@@ -480,14 +483,63 @@ def team_19_adv():
...
+    #     number = int(float(input("Which number do you pick? (only put whole numbers): ")))
+    #     sleep(delay)
+    #     if number >= 13:
+    #         print("Congrats! The snake decided to spare you!")
+    #         dead = False
+    #         sleep(delay)
+    #     elif number < 13:
+    #         print("You chose a wrong number, which angers the snake. He and the other snakes and spiders bite you!")
+    #     else:
+    #         print("Oh no! You died! Try again by hitting the green play button.")
```
Comment (sheggen) [inline t04_refactored.py:541]: Remove old code before PR'ing.

**Comment by collinsc2** [comment]
Moved up docstring and deleted commented lines

File: t04_refactored.py [hunk 10]
```diff
@@ -533,14 +536,48 @@ def team_19_adv():
...
+        print("A bunch of snakes and spiders fall on you!")
+        dead = True
+        sleep(delay)
+    else:
+        # neutral choice
+        print("You don't pick up either of the statues.")
+        sleep(delay)
+        print("Nothing happens... You are still stuck in the cave.")
+        sleep(delay)
+kill_if_dead(dead)
```
Comment (sheggen) [inline t04_refactored.py:580]: This is outside of the function. Indent it one (or two) lines to put it inside the function and/or inside the `else` statement.

**Pull request merged**